webpack.config.js

```diff
@@ -40,32 +40,45 @@ const projectPath = ( relativePath ) => {
 	return path.resolve( fs.realpathSync( process.cwd() ), relativePath );
 };
 
+const seed = {};
+const manifestArgs = {
+	fileName: path.resolve( __dirname, 'dist/manifest.php' ),
+	seed,
+	serialize( manifest ) {
+		const maxLen = Math.max(
+			...Object.keys( manifest ).map( ( key ) => key.length )
+		);
+		const content = manifestTemplate.replace(
+			'{{assets}}',
+			Object.keys( manifest )
+				.map(
+					( key ) =>
+						`"${ key
+							.replace( '.js', '' )
+							.replace( '.css', '' ) }"${ ''.padEnd(
+							maxLen - key.length,
+							' '
+						) } => "${ manifest[ key ] }",`
+				)
+				.join( '\n\t\t' )
```

Comment:
Each line should only have a single indentation.
```suggestion
				.join( '\n\t' )
```

Reply:
https://github.com/google/site-kit-wp/pull/4095/commits/dc71e03de34fb6360f90c28611d8c4f0bd5ddcf2